front/src/components/CTF/LogoLink.vue

```diff
@@ -1,14 +1,23 @@
 <template>
   <q-btn
-    :href="ctf.ctfUrl"
+    :href="ctf.ctfUrl || null"
```

Comment:
Maybe adding a `v-if="ctf.ctfUrl"` to the `q-btn` and not adding `|| null` would make more sense? So just don't display the button if there is no CTF URL set, since the button would not do anything. On the other hand, it would make the design more inconsistent. You can decide yourself what you prefer.

Reply:
I prefer keeping the logo for consistency.